automatic/glmixer/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName            = 'glmixer'
+  fileType               = 'exe'
+  url                    = 'https://sourceforge.net/projects/glmixer/files/Windows%20Binary/GLMixer_1.5-1329_Windows.exe'
```

Comment:
if the software doesn't support 32bit, you could just set this to `''`, it would then fail if the user is running on 32bit or using the `--x86` parameter